workletize-code.js

```diff
@@ -0,0 +1,22 @@
+const process = require('process');
+const babel = require('@babel/core');
+const plugin = require('./node_modules/react-native-reanimated/plugin');
```

Comment:
Just a js file that uses the official reanimated plugin, this process will receive the clojure compiled code and will workletize it